### PR TITLE
Display app versions

### PR DESCRIPTION
**Pull request by rodionovd**
Re: #43

### CODE REVIEW

File: mas-cli/Commands/Outdated.swift [hunk 1]
```diff
@@ -14,8 +14,13 @@ struct OutdatedCommand: CommandType {
     func run(options: Options) -> Result<(), MASError> {
         let updateController = CKUpdateController.sharedUpdateController()
         let updates = updateController.availableUpdates()
+        let softwareMap = CKSoftwareMap.sharedSoftwareMap()
         for update in updates {
-            print("\(update.itemIdentifier) \(update.title) (\(update.bundleVersion))")
+            if let installed = softwareMap.productForBundleIdentifier(update.bundleID) {
```
Comment (argon) [inline mas-cli/Commands/Outdated.swift]: Very nice. This has made me realise a better way to do the multi-upgrade "is installed" checks

File: mas-cli/Commands/Outdated.swift [hunk 2]
```diff
@@ -14,8 +14,13 @@ struct OutdatedCommand: CommandType {
...
+                print("\(update.itemIdentifier) \(update.title) (\(installed.bundleVersion) -> \(update.bundleVersion))")
+            } else {
+                print("\(update.itemIdentifier) \(update.title) (\(update.bundleVersion))")
```
Comment (argon) [inline mas-cli/Commands/Outdated.swift]: I would like to see some kind of indicator that this update is pending even though the app store can't find the currently installed version (seems like it shouldn't happen)

Perhaps:

```swift
print("\(update.itemIdentifier) \(update.title) (unknown -> \(update.bundleVersion))")
```

Comment (rodionovd) [inline mas-cli/Commands/Outdated.swift]: Yep, this situation is unlikely to happen, but let's make it "unknown" just in case

Comment (rodionovd) [inline mas-cli/Commands/Outdated.swift]: OK, done

**Pull request merged**